src/libraries/Microsoft.PowerFx.Core/Public/DefinitionsCheckResult.cs

```diff
@@ -49,9 +51,10 @@ public class DefinitionsCheckResult : IOperationStatus
         // All errors accumulated. 
         private readonly List<ExpressionError> _errors = new List<ExpressionError>();
 
-        public DefinitionsCheckResult()
+        public DefinitionsCheckResult(Features features = null)
         {
             _localSymbolTable = new SymbolTable { DebugName = "LocalUserDefinitions" };
+            _features = features ?? Features.None;
```

Comment:
> Features.None; [](http://example.com/codeflow?start=35&length=15)

Can we default to v1? And let PA fall back to v0?

Reply:
using V1 for Definitions but still keeping None for Format and other uses, as TexlParser defaults to None